components/playlist/common/mojom/playlist.mojom

```diff
@@ -73,6 +73,7 @@ struct PlaylistItem {
   // Playlists that contain this item. Could have multiple |Playlist| in case
   // user copied this item to another list.
   array<string> parents;
+  bool is_selected;
```

Comment:
Can you make this [EnableIf=is_android] too? I think that would be better to prevent side effect on desktop side.